src/store/tests/test-data/cat-b/vehicle-checks/vehicle-checks.reducer.ts

```diff
@@ -0,0 +1,86 @@
+import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
+import { CompetencyOutcome } from 'src/app/shared/models/competency-outcome';
```

Comment:
can be changed to '@shared...' with new path alias you have added

Reply:
changed